### PR TITLE
GDPR check

### PR DESCRIPTION
**Pull request by tomlovesgithub**
## What's included?
An additional page - akin to the eligibility check prior to applying to an exercise - with a GDPR statement and agreement.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to apply to any exercise.
- Go past eligibility check.
- Observe GDPR page.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context
image
image

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

File: src/views/Eligibility/EligibilityFail.vue [hunk 1]
```diff
@@ -39,8 +39,8 @@ export default {
     isSignedIn() {
       return this.$store.getters['auth/isSignedIn'];
     },
-    applyLink() {
-      return this.isSignedIn ? `/apply/${this.vacancy.id}` : { name: 'sign-in' };
+    GDPRLink() {
```
Comment (warrensearle) [inline src/views/Eligibility/EligibilityFail.vue:42]: Sorry, this is a picky one!
You could probably call this `continueLink` say as it doesn't always go to the GDPR page.
I would expect a method like `GDPRLink` to always return the link to GDPR page.

**Review by warrensearle — COMMENTED** [review]
If you are not signed in and go through the `Vacancy details -> Eligibility check -> Sign In` path you miss the GDPR page. Is that intended?

File: src/views/GDPR/GDPRCheck.vue [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  computed: {
+    vacancy() {
+      return this.$store.state.vacancy.record;
+    },
+    isSignedIn() {
+      return this.$store.getters['auth/isSignedIn'];
+    },
+    ContinueLink() {
+      return this.isSignedIn ? { name: 'GDPR' } : { name: 'sign-in' };
+    },
```
Comment (warrensearle) [inline src/views/GDPR/GDPRCheck.vue:69]: Is this used anywhere in this view?

**Review by warrensearle — APPROVED** [review]
All working @tomlovesgithub 👍 

There's possibly one redundant piece of code. Please check

**Review by HalcyonJAC — APPROVED** [review]
Code looks great to me!

**Comment by tomlovesgithub** [comment]
- bullet points.

**Pull request merged**